Error when building with webpack

```
ERROR in ./~/@horizon/client/lib/util/fetch.js
Module not found: Error: Cannot resolve module 'exports' in ./node_modules/@horizon/client/lib/util
 @ ./~/@horizon/client/lib/util/fetch.js 13:0-69
```

Server Version: 1.0.1
Client Version: 1.0.1

Installed client with `npm install @horizon/client`

I am following the second option in the instructions for [Integrating Horizon](http://horizon.io/docs/getting-started/#integrating-horizon-with-an-existing-application). 

I'm seeing the error above. I installed `@horizon/client` and attempted to build it within my existing setup. Apparently this has something to do with the imports loader. Installing `imports-loader` in my project doesn't fix the problem.

In fetch.js:

```
require('imports?this=>global!exports?global.fetch!isomorphic-fetch');
```

That is the problem line.

My code is exactly the example on the [`@horizon/client`](https://www.npmjs.com/package/@horizon/client#example) npm page:

```
const Horizon = require("@horizon/client")
const horizon = Horizon()
```

@deontologician Yes. Let me give you some more detailed information.

I have a simple project setup to build with webpack.

Here is my webpack config file:

```
const path = require('path');

const CopyWebpackPlugin = require('copy-webpack-plugin');
const autoprefixer = require('autoprefixer');

const DIST = path.resolve(__dirname, 'dist');

module.exports = {
  entry: {
    app: './app/index.js',
  },
  devServer: {
    outputPath: DIST,
  },
  output: {
    path: DIST,
    filename: '[name].js',
    publicPath: '/',
  },
  module: {
    loaders: [
      {
        test: /\.jsx?$/,
        exclude: /(node_modules|bower_components)/,
        loaders: ['babel-loader'],
      },
      {
        test: /\.scss$/,
        loaders: ['style', 'css?modules&importLoaders=1&sourceMap!postcss!sass?sourceMap&sourceMapContents'],
      },
      {
        test: /\.json$/,
        exclude: /(node_modules|bower_components)/,
        loader: 'json-loader',
      },
    ],
  },
  plugins: [
    new CopyWebpackPlugin([
      {from: 'app/index.html'},
    ]),
  ],
  resolve: {
    extensions: ['', '.js', '.jsx'],
  },
  devtool: 'inline-source-map',
  postcss: [autoprefixer],
};
```

Webpack essentially builds the application bundle, places it in the `dist` directory and also copies a simple `index.html` file to the same directory. These are both then to be served by Horizon.

My `app/index.js` file is very simple too:

```
const Horizon = require("@horizon/client")
const horizon = Horizon()

horizon.onReady(() => {
  document.querySelector('h1').innerHTML = 'api works!'
});
horizon.connect();
```

I copied this code from the index.html file generated and used the `require()` from the docs I linked to above.

The problem is that when I run my webpack build, I get this error:

```
ERROR in ./~/@horizon/client/lib/util/fetch.js
Module not found: Error: Cannot resolve module 'exports' in ./node_modules/@horizon/client/lib/util
 @ ./~/@horizon/client/lib/util/fetch.js 13:0-69
```

This error is not in my code. It has to do with a line in fetch.js which is using the webpack `imports-loader` package. 

This is a bug because I can't build using `require('@horizon/client')` due to this error.

I get a similar error with webpack, but mine reads "imports" instead of "exports"
it was working with version 0.3.2
now i am on version horizon v1.0.1
ERROR in ./~/@horizon/client/lib/util/fetch.js
Module not found: Error: Cannot resolve module **'imports'** in /Users/sachin/sites/node/h7/node_modules/@horizon/client/lib/util
 @ ./~/@horizon/client/lib/util/fetch.js 13:0-69

My webpack config is even simpler
{
  "name": "H7",
  "version": "0.0.1",
  "description": "Horizon VueJs Webpack Starter",
  "dependencies": {
    "vue": "^1.0.24",
    "@horizon/client": "^1.0.1"
  },
  "devDependencies": {
    "babel-core": "^6.1.2",
    "babel-loader": "^6.1.0",
    "babel-plugin-transform-runtime": "^6.1.2",
    "babel-preset-es2015": "^6.1.2",
    "babel-preset-stage-0": "^6.1.2",
    "babel-runtime": "^5.8.0",
    "webpack": "^1.13.0",
    "css-loader": "^0.23.0",
    "style-loader": "^0.13.0",
    "vue-loader": "^7.3.0",
    "vue-html-loader": "^1.0.0",
    "vue-hot-reload-api": "^1.2.0"
  },
  "scripts": {
    "dev": "hz serve --dev & webpack-dev-server --inline --hot"
  },
  "author": "Sachin"
}

--- a/client/package.json
+++ b/client/package.json
@@ -18,6 +18,8 @@
     "core-js": "^2.1.0",
     "engine.io-client": "^1.6.9",
     "engine.io-parser": "socketio/engine.io-parser#748144b50a1d10e8c7c9b8100f2e21f8ac424c7a",
+    "exports-loader": "^0.6.3",
+    "imports-loader": "^0.6.5",
     "isomorphic-fetch": "^2.2.1",
     "rxjs": "^5.0.0-beta.6",
     "snake-case": "^1.1.2"
@@ -39,8 +41,6 @@
     "copy-webpack-plugin": "^1.1.1",
     "cross-env": "^1.0.7",
     "eslint": "2.2.x",
-    "exports-loader": "^0.6.3",
-    "imports-loader": "^0.6.5",
     "istanbul": "^0.4.2",
     "lodash": "^4.6.1",
     "mocha": "2.3.4",

